src/core_plugins/metrics/public/components/data_format_picker.js

```diff
@@ -73,39 +87,51 @@ class DataFormatPicker extends Component {
       { label: 'Duration', value: 'duration' },
       { label: 'Custom', value: 'custom' }
     ];
+    const selectedOption = options.find(option => {
+      return defaultValue === option.value;
+    });
 
     let custom;
     if (defaultValue === 'duration') {
       const [from, to, decimals] = value.split(',');
+      const selectedFrom = durationInputOptions.find(option => {
+        return from === option.value;
+      });
+      const selectedTo = durationInputOptions.find(option => {
+        return to === option.value;
+      });
       return (
         <div className="vis_editor__data_format_picker-container">
           <div className="vis_editor__label">
             {this.props.label}
           </div>
           <div className="vis_editor__item">
-            <Select
-              clearable={false}
-              value={defaultValue}
+            <EuiComboBox
+              isClearable={false}
               options={options}
+              selectedOptions={selectedOption ? [selectedOption] : []}
               onChange={this.handleChange}
+              singleSelection={true}
             />
           </div>
           <div className="vis_editor__label">From</div>
           <div className="vis_editor__item">
-            <Select
-              clearable={false}
-              value={from}
+            <EuiComboBox
+              isClearable={false}
               options={durationInputOptions}
+              selectedOptions={selectedFrom ? [selectedFrom] : []}
               onChange={this.handleDurationChange('from')}
+              singleSelection={true}
             />
           </div>
           <div className="vis_editor__label">To</div>
           <div className="vis_editor__item">
-            <Select
-              clearable={false}
-              value={to}
-              options={durationOutputOptions}
+            <EuiComboBox
+              isClearable={false}
+              options={durationInputOptions}
```

Comment:
Any reason why you changed this from outputOptions to inputOptions? Looks like previously we didn't want pico,nano or micro as an output option (no idea why...)

Reply:
This was removed by accident. I will replace with the original options list.